nestkernel/parameter.h

```diff
@@ -61,6 +65,11 @@ class Parameter
   {
```

Comment:
This constructor seems odd since it does nothing, it does not even set `parameter_is_spatial_` or `parameter_returns_int_only_`.

Reply:
True, I have removed it.